src/scenes/Event/components/HistoryTable/styles.js

```diff
@@ -0,0 +1,18 @@
+const styles = (theme) => ({
+  pickerTab: {
+    margin: 0,
+    marginTop: 0,
+    paddingTop: 10,
+    padding: 10,
```

Comment:
remove all margin and padding. we already define `MuiTab` in global theme overrides so lets use the default version to handle all the margins.